src/main/scala/devices/debug/Debug.scala

```diff
@@ -563,6 +608,7 @@ class TLDebugModuleOuter(device: Device)(implicit p: Parameters) extends LazyMod
     // hartsel/hasel/hamask must also be used by the DebugModule state machine,
     // so it is passed to Inner.
 
+    // sends debug interruption to Core when dmcs.haltreq is set,
```

Comment:
Maybe merge with the comment block above?

Reply:
In fact, the comment block above has nothing to do with Interrupt mechanism. So I'd rather keep this line and move the block above to where it belongs.